src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Styles/ColorsSamplePage.xaml

```diff
@@ -289,6 +289,140 @@
 			<sample:SamplePageLayout.CupertinoTemplate>
 				<DataTemplate>
 					<StackPanel>
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup>
+
+								<VisualState x:Name="Medium">
+									<VisualState.StateTriggers>
+										<AdaptiveTrigger MinWindowWidth="725" />
+									</VisualState.StateTriggers>
+								</VisualState>
+
+								<VisualState x:Name="Narrow">
+									<VisualState.StateTriggers>
+										<AdaptiveTrigger MinWindowWidth="0" />
+									</VisualState.StateTriggers>
+
+									<VisualState.Setters>
+
+										<Setter Target="Content_1.(Grid.Column)"
+												Value="0" />
+										<Setter Target="Content_1.(Grid.Column)"
+												Value="0" />
+										<Setter Target="Content_1.(Grid.Row)"
+												Value="0" />
+
+										<Setter Target="Content_2.(Grid.Column)"
+												Value="0" />
+										<Setter Target="Content_2.(Grid.Row)"
+												Value="1" />
+
+										<Setter Target="Column_2.Width"
+												Value="0" />
+										<Setter Target="ColumnSpacer_1.Width"
+												Value="0" />
+										<Setter Target="ColumnSpacer_2.Width"
+												Value="0" />
+
+										<Setter Target="TitleContainer_1.MinHeight"
+												Value="0" />
+										<Setter Target="TitleContainer_2.MinHeight"
+												Value="0" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+
+						<!-- TODO : Replace white foreground with proper StaticResource  -->
+						<ScrollViewer VerticalScrollBarVisibility="Auto">
+							<Grid>
+								<Grid.ColumnDefinitions>
+									<ColumnDefinition x:Name="Column_1"
+													  Width="*" />
```

Comment:
Indentation seems off for this file

Reply:
I cannot seem to change the identation on my side. Nothing happens after ctrl+K+D or if I change it manually